src/app/pages/about-us/components/rating/rating.component.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, HostBinding, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-rating',
+  templateUrl: './rating.component.html',
+  styleUrls: ['./rating.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RatingComponent {
+  @Output()
+  readonly changeRate = new EventEmitter<number>();
+
+  @HostBinding('class')
+  private readonly classes = 'flex mt-2 mb-5';
+
+  currentRate: number;
+  selectedStarIndex: number;
+
```

Comment:
Place public fields and private in correct order.
Now you have here: public, private, and again public